frontend/src/i18n/locales/en.ts

```diff
@@ -91,6 +91,9 @@ export default {
     },
     subject: {
         register: "Register to subject:",
+        ok: "Ok",
+        cancel: "Cancel",
```

Comment:
```suggestion
        register: "You have been invited to join",
        ok: "Accept",
        cancel: "Cancel",
```
Iets in deze zin, ik vind de originele verwoording een beetje barebones.

In het algemeen ook duidelijker als de knop zegt wat hij doet.